dataflux_pytorch/benchmark/README.md

```diff
@@ -52,26 +52,25 @@ $ python dataflux_pytorch/benchmark/lightning_checkpoint_benchmark.py
+.venv/lib/python3.12/site-packages/lightning/pytorch/trainer/connectors/logger_connector/logger_connector.py:75: Starting from v1.9.0, `tensorboardX` has been removed as a dependency of the `lightning.pytorch` package, due to potential conflicts with other packages in the ML ecosystem. For this reason, `logger=True` will use `CSVLogger` as the default logger, unless the `tensorboard` or `tensorboardX` packages are found. Please `pip install lightning[extra]` or one of them to enable TensorBoard support by default
 
   | Name  | Type        | Params | Mode
 ----------------------------------------------
-0 | model | Transformer | 19.8 M | train
+0 | model | Transformer | 658 M  | train
 ----------------------------------------------
-19.8 M    Trainable params
+658 M     Trainable params
 0         Non-trainable params
-19.8 M    Total params
-79.189    Total estimated model params size (MB)
-/usr/local/google/home/divyarawal/dataflux_dev/.venv/lib/python3.12/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:424: The 'train_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=47` in the `DataLoader` to improve performance.
-Epoch 0:   0%|                                                                                                               | 5/59674 [00:11<39:37:59,  0.42it/s, v_num=2]`Trainer.fit` stopped: `max_steps=5` reached.
-Epoch 0:   0%|                                                                                                               | 5/59674 [00:11<39:38:05,  0.42it/s, v_num=2]
-Time to train over 5 steps: 14.197517395019531 seconds
-Time to save one checkpoint: 2.075364589691162 seconds
+658 M     Total params
+2,634.181 Total estimated model params size (MB)
+.venv/lib/python3.12/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:424: The 'train_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=47` in the `DataLoader` to improve performance.
+Epoch 0:   0%|                                                                                                            | 10/59674 [12:17<1221:29:06,  0.01it/s, v_num=5]`Trainer.fit` stopped: `max_steps=10` reached.
+Epoch 0:   0%|                                                                                                            | 10/59674 [12:17<1221:29:09,  0.01it/s, v_num=5]
+Average time to save one checkpoint: 58.68560411930084 seconds
 ```
 
 ## Results
 
-The table below contains benchmarking times writing checkpoints to GCS.
+The table below contains benchmarking times writing checkpoints to GCS, the average save time is taken over 10 steps.
```

Comment:
ha so we are now averaging 10 writes now and we are still seeing slowness. As we discussed, have you checked the tput on the bucket?

Reply:
With the average we do still see the slow down. I updated the table with the average write speeds. I am looking into the dashboard. The ingress is looking similar for the 14GB file I am writing, but the times are not the same (blobwriter still slower). Testing a couple more file sizes for data points. All of this will be in a document shared out.